src/cmd-create-legacy-oscontainer

```diff
@@ -126,6 +134,14 @@ with open(digestfile) as f:
     osc_digest = f.read().strip()
 
 # Inject the oscontainer with SHA256 into the build metadata
+sha256 = hashlib.sha256()
+with open(f"{latest_build_path}/{args.name}", "rb") as f:
+    for b in iter(lambda: f.read(4096), b""):
+        sha256.update(b)
+size_bytes = os.path.getsize(f"{latest_build_path}/{args.name}")
+meta['images']['legacy-oscontainer'] = {'path': args.name,
+                                        'sha256': sha256.hexdigest(),
+                                        'size': size_bytes}
```

Comment:
We might need to make sure `cosa compress` doesn't try to compress this artifact.

Reply:
yes. see what we do for the `ostree` artifact: https://github.com/coreos/coreos-assembler/blob/3d742f1195e1d6ad892538f54c184ea5b1cfa625/src/cmd-build#L473